pkg/private/tar/tar_writer.py

```diff
@@ -107,6 +108,8 @@ def __init__(self,
     self.directories.add('/')
     self.directories.add('./')
 
+    self.create_parents = create_parents
```

Comment:
This does not need another blank line above it.

Reply:
Acknowledged.